scripts/flattened-migrate.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable camelcase */
+/* eslint-disable new-cap */
+
+const exec = require("child_process").exec
+const fs = require("fs")
+const os = require("os")
+const cli = new cli_func()
+
+if (process.argv.length < 4) {
+  console.log()
+  console.log("Usage: npm|yarn migrate:flattened <ArtifactName> <Network>")
+  process.exit(0)
+}
+
+const artifact = process.argv[2]
+const network = process.argv[3]
+process.env.FLATTENED_DIRECTORY = `./contracts/flattened/${artifact}/`
+
+if (!fs.existsSync(`${process.env.FLATTENED_DIRECTORY}/Flattened${artifact}.sol`)) {
+  console.log()
+  console.log("> Please, flatten the artifact first. E.g.:")
+  console.log(`  $ yarn flatten contracts${os.type() === "Windows_NT" ? "\\" : "/"}${artifact}.sol`)
+  console.log()
```

Comment:
It's a personal preference: :smile: 

```suggestion
console.log("\n\
    > Please, flatten the artifact first. E.g.:\n\
       $ yarn flatten contracts${os.type() === "Windows_NT" ? "\\" : "/"}${artifact}.sol`)
```